pyproject.toml

```diff
@@ -45,31 +45,31 @@ testpaths = "cf_units"
 write_to = "cf_units/_version.py"
 local_scheme = "dirty-tag"
 
-[tool.black]
+[tool.ruff]
 line-length = 79
-target-version = ["py39", "py310", "py311"]
-include = '\.pyi?$'
-exclude = '''
-(
-  /(
-      \.eggs         # exclude a few common directories in the
-    | \.git          # root of the project
-    | \.hg
-    | \.mypy_cache
-    | \.tox
-    | \.venv
-    | _build
-    | buck-out
-    | build
-    | dist
-  )/
-  | _version.py
-)
-'''
+target-version = "py38"
+select = [
+    # pyflakes
+    "F",
+    # pycodestyle
+    "E",
+    "W",
+    # flake8-bugbear
+    "B",
+    # flake8-comprehensions
+    "C4",
+    # isort
+    "I",
+    # pyupgrade
+    "UP",
+]
+ignore = ["B904", "F403", "F405"]
```

Comment:
Don't know that these are reasonable? Ideally, they wouldn't be ignored, IMO. Perhaps have more specific ignores for files.

This can be done in the future - no need for this PR.

Reply:
Yep, that has been the approach in Iris: get ruff in with rules skipped and then de-skip in a series of follow-ups.